OmiseSwiftTests/ChargesOperationFixtureTests.swift

```diff
@@ -1137,7 +1137,54 @@ class ChargesOperationFixtureTests: FixtureTestCase {
         XCTAssertEqual(items[4].name, "metadata[customer_id]")
         XCTAssertEqual(items[4].value, "123")
     }
+
+    func testSCBMobileBankingChargeCreate() {
+        let expectation = self.expectation(description: "Mobile Banking SCB Charge create")
+        
+        let createParams = ChargeParams(value: Value(amount: 100_00, currency: .thb),
+                                        sourceType: .mobileBanking(.scb),
+                                        returnURL: defaultReturnURL)
+        
+        Charge.create(using: testClient, params: createParams) { (result) in
+            defer { expectation.fulfill() }
+            
+            switch result {
+            case let .success(charge):
+                XCTAssertNotNil(charge)
+                XCTAssertEqual(charge.value.amount, 10_000_00)
+                XCTAssertEqual(charge.source?.paymentInformation, .mobileBanking(.scb))
+                XCTAssertEqual(charge.returnURL, defaultReturnURL)
+            case let .failure(error):
+                XCTFail("\(error)")
+            }
+        }
+        
+        waitForExpectations(timeout: 15.0, handler: nil)
+    }
     
+    func testSCBMobileBankingChargeRetrieve() {
+            let expectation = self.expectation(description: "Mobile Banking Charge result")
+            
+            let request = Charge.retrieve(using: testClient, id: "chrg_test_5m7f02ng1lxmr8ryb2a") { (result) in
+                defer { expectation.fulfill() }
+                
+                switch result {
+                case let .success(charge):
+                    XCTAssertEqual(charge.value.amount, 100)
+                    XCTAssertEqual(charge.status, .pending)
+                    XCTAssertEqual(charge.source?.paymentInformation.sourceType,EnrolledSource.EnrolledPaymentInformation.mobileBanking(.scb).sourceType)
+                    XCTAssertEqual(charge.source?.flow, .appRedirect)
+                    XCTAssertEqual(charge.authorizedURL,URL(string:"http://lvh.me:52000/confirm_test?payload=pj%25C1ak%259C%25D44%25C2%2526%25CE%25CA%2584%2513%25D2%25BF%25F1%25DE%257B%25ED%2529_%250A%257Bu%25E3%2504%25D1%25A8%25C0%25BF%25A9%25E4%25AD%259F%2587%25BBh%259F%2589%25C2%25C5G%25B6U%25B8%258D%25C1%252B%259B%25BEe%25F1%25E7%253FQ%250C8%2501%251F%258B%2599%25094%2512%25CB%25DD%25E5i%25E8%25CD%255D%25A1%25A9%25F5%25E0%2599%250D%2502%2580%25F4%253C%250C%2505%250B%25C6l%2529%2523Q%25CC%253D%25C0%2582Z%251A%25B1%259D%257F%25B1%255E%25A5%2591%2589%25D6%25DEU%25AB%25A1%25CC%253BS%25E7hY%25BB%25A0%25A1hp%2523%25E2%250Ee%25F2l%252B%2525"))
+                    XCTAssertEqual(charge.returnURL, URL(string:"scbeasysim://purchase/811bbdf9-7255-4d4b-af20-002848c5e84b"))
```

Comment:
I tried to run the test and it fails. The `authorizedURL` is nil and the `returnURL` is `http://www.google.com`. Can you check what's the issue there?

Reply:
On it 🙏